src/wasm.ts

```diff
@@ -133,7 +133,11 @@ export async function buildWasm(
         (responseText.length + 1) * Int8Array.BYTES_PER_ELEMENT
       );
       if (responseAddress === 0) {
-        throw new Error('malloc failed: Cannot allocate memory');
+        pvError?.addError('pvHttpsRequestWasm', "malloc failed: Cannot allocate memory");
```

Comment:
I think the memory errors should probably all have the same name having to do with memory. The message can specify more information about what failed to alloc